tests/unit/core/containers/TestServerHtml.js

```diff
@@ -113,7 +114,7 @@ describe('<ServerHtml />', () => {
   it('renders favicon', () => {
     const html = findRenderedDOMComponentWithTag(render(), 'html');
     const favicon = html.querySelector('link[rel="shortcut icon"]');
-    expect(favicon.getAttribute('href')).toEqual('/favicon.ico?v=1');
+    expect(favicon.getAttribute('href')).toEqual(`${config.get('amoCDN')}/favicon.ico`);
```

Comment:
As is, this test relies on the [test configuration](https://github.com/mozilla/addons-frontend/blob/master/config/test.js) value of `amoCDN`. In practice that's fine but I think it would make the test more robust if you relied on something explicit. This will help with maintenance as well as with understanding the intention of the test. Here's an example of how it would look:

```js
import { getFakeConfig } from 'tests/unit/helpers';
...
const amoCDN = 'https://someserver.net';
const _config = getFakeConfig({ amoCDN });

const html = findRenderedDOMComponentWithTag(render({ _config }), 'html');
expect(favicon.getAttribute('href')).toEqual(`${amoCDN}/favicon.ico`);
```

This change requires you to set up `_config` as a property on `<ServerHtml>` for dependency injection. Here's an example:

```js
export class SomeComponent extends React.Component {
  static PropTypes = {
    _config: PropTypes.object,
  };

  static defaultProps = {
    _config: config,
  }

  ...
}
```

You then need to change your access of `config.get('amoCDN')` to something like:

```js
const { _config } = this.props;
...
_config.get('amoCDN');
```